app/src/main/java/com/sdpteam/connectout/mapList/MapListViewModel.java

```diff
@@ -0,0 +1,30 @@
+package com.sdpteam.connectout.mapList;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+import com.sdpteam.connectout.event.Event;
+
+import java.util.List;
+
+public class MapListViewModel extends ViewModel {
```

Comment:
MapList as said is not a good name, what is it doing a GoogleMaps Map or a List view or... 
The ViewModel is going to be the same for Map View and List View modes. Maybe call it just `EventsViewModel`